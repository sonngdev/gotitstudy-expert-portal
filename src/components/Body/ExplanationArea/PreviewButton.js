import React from 'react';
import { ButtonWithModal } from '../../shared';

export function PreviewButton() {
  return (
    <ButtonWithModal
      buttonProps={{
        className: 'flag-session',
        variant: 'positive',
      }}
      buttonAction="Preview â–¶"
      modalSize="medium"
      modalTitle="Explanation preview"
      modalBody="Hello"
      confirmButtonText="Submit"
      onConfirm={() => window.location.reload()}
    />
  );
}
