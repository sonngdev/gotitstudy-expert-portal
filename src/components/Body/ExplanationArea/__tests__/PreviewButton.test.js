import React from 'react';
import { shallow } from 'enzyme';
import { PreviewButton } from '../PreviewButton';

describe('<PreviewButton />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<PreviewButton />);
  });

  it('renders', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
