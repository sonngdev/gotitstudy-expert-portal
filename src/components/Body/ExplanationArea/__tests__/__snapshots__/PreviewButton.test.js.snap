// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviewButton /> renders 1`] = `"<ButtonWithModal buttonProps={{...}} buttonAction=\\"Preview â–¶\\" modalSize=\\"medium\\" modalTitle=\\"Explanation preview\\" modalBody=\\"Hello\\" confirmButtonText=\\"Submit\\" onConfirm={[Function: onConfirm]} />"`;
